web/crux-ui/e2e/with-login/dashboard.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { ROUTE_DASHBOARD, ROUTE_PROFILE } from '@app/routes'
+import { expect, test } from '@playwright/test'
+
+test.describe('Onboarding', () => {
+  test('should be toggleable', async ({ page }) => {
+    await page.goto(ROUTE_DASHBOARD)
+    await page.waitForURL(ROUTE_DASHBOARD)
+    await expect(page.locator('label:has-text("Onboarding")')).toBeVisible()
+
+    await page.goto(ROUTE_PROFILE)
+    await page.locator('button[role="switch"]').click()
```

Comment:
I think this should also look for the label. If there are multiple toggles on the profile page this won't work anymore.